mailpoet/assets/css/src/components-plugin/_forms.scss

```diff
@@ -75,8 +75,13 @@ progress::-moz-progress-bar {
   justify-content: space-between;
   margin: $grid-gap auto;
 
+  .mailpoet-form-send-email & {
+    grid-template-columns: $grid-column $grid-column;
+    max-width: 976px;
+  }
+
```

Comment:
> I decided not to use "mailpoet-main-container" because it's too wide, for two 368px columns.

:+1:, but do you think this is a good place to fix it (plus the `!important`)? I would argue that a form from "components" doesn't need to know about a specific email form. What I mean is that rather than `.mailpoet-form-grid { .mailpoet-form-send-email & { ... } }` here we could do `.mailpoet-form-send-email { .mailpoet-form-grid { ... } }` in another place.

Or maybe even easier, add a new `.mailpoet-form-send-email-container`...